narwhals/__init__.py

```diff
@@ -31,22 +31,22 @@
 from narwhals.dtypes import UInt64
 from narwhals.dtypes import Unknown
 from narwhals.expr import Expr
-from narwhals.expr import all_ as all  # noqa: A004
+from narwhals.expr import all_ as all
```

Comment:
Ok something must have gone wrong with the pre-commit sync :|

Reply:
🤔 i'm so confused 😄